File: Java_Course/src/App/BasicCalculator.java
package App;

import java.util.Scanner;

public class BasicCalculator {
	public static void basicCalculator(String [] args) {
		
		Scanner keyboardInput = new Scanner(System.in);
		
		System.out.print("Enter the first number: ");
		double num1 = keyboardInput.nextDouble();
		
		System.out.print("Enter the second number: ");
		double num2 = keyboardInput.nextDouble();
		
		double result = num1 + num2;
		System.out.println("The result is: " + result);
		
		
		System.out.println();
	}
}

File: php_Course/www/basicCalculator.php
<!DOCTYPE html>
<html lang="en" dir="ltr">
  <head>
    <meta charset="utf-8">
    <title></title>
  </head>
  <body>

    <form action="basicCalculator.php" method="get">
      First Number: <input type="number" name="num1" value="0">
      <br>
      Second Number: <input type="number" name="num2" value="0">
      <input type="submit">
    </form>

    Answer: <?php echo $_GET["num1"] + $_GET["num2"]?>

  </body>
</html>
